docs/turn-howto.md

```diff
@@ -42,65 +38,112 @@ This will install and start a systemd service called `coturn`.
 
         ./configure
 
-    You may need to install `libevent2`: if so, you should do so in
-    the way recommended by your operating system. You can ignore
-    warnings about lack of database support: a database is unnecessary
-    for this purpose.
+    > You may need to install `libevent2`: if so, you should do so in
+    > the way recommended by your operating system. You can ignore
+    > warnings about lack of database support: a database is unnecessary
+    > for this purpose.
 
 1.  Build and install it:
 
         make
         make install
 
+#### Docker installation
+There are coturn docker images available. You can prepare by running
+
+        docker pull instrumentisto/coturn
```

Comment:
https://github.com/spantaleev/matrix-docker-ansible-deploy [recently](https://github.com/spantaleev/matrix-docker-ansible-deploy/pull/1009) changed to the [official coturn image](https://hub.docker.com/r/coturn/coturn/). It looks like few, if any, changes are required. could we do the same here?